File: src/main/java/com/nm/Militaryoffice/repository/mapper/CommissionDecisionMapper.java
package com.nm.Militaryoffice.repository.mapper;

import com.nm.Militaryoffice.model.CommissionDecision;
import org.springframework.jdbc.core.RowMapper;

import javax.swing.tree.TreePath;
import java.sql.ResultSet;
import java.sql.SQLException;

public class CommissionDecisionMapper implements RowMapper<CommissionDecision> {

    @Override
    public CommissionDecision mapRow(ResultSet rs, int rowNum) throws SQLException {
        var decision = new CommissionDecision();
        decision.setDecision_id(rs.getLong("decision_id"));
        decision.setId(rs.getLong("id"));
        decision.setDecision_date(rs.getDate("decision_date"));
        decision.setDecision_text(rs.getString("decision_text"));
        decision.
                setOrganisation_that_made_decision(rs.
                        getString("organisation_that_made_decision"));
        decision.setCommissioner_fullname(rs.getString("commissioner_fullname"));
        decision.setIs_enlistment(rs.getBoolean("is_enlistment"));
        decision.setIs_conscript_signed(rs.getBoolean("is_conscript_signed"));

        return decision;
    }
}

File: src/main/java/com/nm/Militaryoffice/repository/mapper/CommissionDecisionMapper.kt
package com.nm.Militaryoffice.repository.mapper

import com.nm.Militaryoffice.model.CommissionDecision
import org.springframework.jdbc.core.RowMapper
import java.sql.ResultSet

class CommissionDecisionMapper: RowMapper<CommissionDecision> {
    override fun mapRow(rs: ResultSet, rowNum: Int): CommissionDecision {
        val decision = CommissionDecision()
        decision.id = rs.getLong("id")
        decision.decision_id = rs.getInt("decision_id")
        decision.decision_text = rs.getString("decision_text")
        decision.organisation_that_made_decision = rs.getString("orgasnisation_that_made_decision")
        decision.commissioner_fullname = rs.getString("commissioner_fullname")
        decision.is_enlistment = rs.getBoolean("is_enlistment")
        decision.is_conscript_signed = rs.getBoolean("is_conscript_signed")
        return decision
    }
}